### PR TITLE
bump used base images to Fedora 31

### PR DESCRIPTION
**Pull request by phoracek**
Signed-off-by: Petr Horacek <phoracek@redhat.com>

### CODE REVIEW

File: cmd/marker/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM fedora:29
+FROM fedora:31
```
Comment (qinqon) [inline cmd/marker/Dockerfile]: We can even use scratch linux, for this images or even ubi8-minimal

File: cmd/plugin/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM fedora:29
+FROM fedora:31
```
Comment (qinqon) [inline cmd/plugin/Dockerfile]: ditto

File: cmd/marker/Dockerfile (hunk 1 above)
Comment (phoracek) [reply cmd/marker/Dockerfile]: If we do use scratch, we won't be able to use even ls to debug. We have to log to the respective node and use nsenter to do any debugging. We also won't be able to sleep. We can try to optimize it even further in another PR, but focus of this one is only to bump to a newer version.

Regarding the ubi8, it does not have openvswitch I'm any of the repositories. We need the RPM for one last call in the plugin. Once we replace the call with a socket (as Sebastian did with all the other functions), I'd be all for moving to ubi8. I opened an issue for that.

Comment (qinqon) [reply cmd/marker/Dockerfile]: ack

**Pull request merged**